server/views/pages/visits/summary.njk

```diff
@@ -141,6 +139,14 @@
               text: visit.prisonerId,
               attributes: { "data-test": "prisoner-number" }
             },
+            {
+              text: visit.firstBookedDateTime | formatDate("d MMMM") +
+                " at " + visit.firstBookedDateTime | formatTime,
+              attributes: {
+                "data-test": "booked-on",
+                "data-sort-value": visit.firstBookedDateTime | formatDate("t")
```

Comment:
Adding a timestamp as `data-sort-value` to make sorting work correctly. Other columns work by default.